src/content/docs/logs/logs-everywhere/ruby-telemetry-context.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: 'Ruby telemetry in context'
+metaDescription: For apps monitored by the Ruby agent, learn how to link log data with related data across the rest of the New Relic platform.
+---
+
+**REVIEWER: My biggest concern is that I'm throwing way too many words into this doc. I can't decide if it's a good idea to focus on saying why this matters, or does it go too far? Would a better approach be to emphasize how simple it is? For example, if it really is just 1-update to the latest agent, and 2-do any optional config you want to do, maybe we can have one doc for all the agents? Please feel free to comment or edit directly in my PR. Thanks!**
```

Comment:
I like including why this matters/how to use it. If something is going to be enabled for me, I want simple direction on how to use it. This is where UI screenshots may also come in handy! 

I think this could work well as one doc for all the agents, as long as there were either per-language links or a dropdown menu to break down the configuration. I imagine users would find it convenient to have the content on this page (though the release notes route could also work!)

Reply:
We can experiment both ways. My goal is to have the fewest number of docs to make it look as easy as possible!